### DISCUSSION TITLE
windows 10 calendar view (clock view) in windows 11 mode

### DISCUSSION THREAD

**Discussion by Mo0odMC** [valinet/ExplorerPatcher#772] [Ideas]
Basically, the old calendar view of windows 10 but with the look and feel of windows 11.
when you click on the date and time to see the calendar, in windows 10 it shows your events (agenda). but currently with explorerpatcher it would restore the look of windows 10. 
i would like the feature to be more like windows 11 style but with windows 10 functionality.

**Comment by krlvm** [comment]
@valinet, I have made a little program which summons calendar flyout, "detaches" it from taskbar and rounds it corners some time ago. If you can call some code when user opens flyout, this might be a good feature to have.

```cpp
#include <Windows.h>
#include <dwmapi.h>

#pragma comment(lib, "dwmapi.lib")

int main() {
        PostMessageW(FindWindowExW(NULL, NULL, L"Shell_TrayWnd", NULL), 1486, 0, 0);
        Sleep(1000);
        HWND hWnd = GetForegroundWindow();

        RECT rc;
        GetWindowRect(hWnd, &rc);

        SetWindowPos(hWnd, NULL, rc.left - 12, rc.top - 12, 0, 0, SWP_NOSIZE);

        int val = 2;
        DwmSetWindowAttribute(hWnd, 33, &val, sizeof(val));

        return 0;
}
```

**Comment by Andre-10** [comment]
Calendar with agenda is one of few thing why I'm using ExporerPatcher with Win 10 Taskbar.
In the meantime, if you want Windows 11 Taskbar - you know that you can set Windows calendar in list View? Just shrink it horizontally, and turn off menu. Looks like this (I have it on my second display...)
![image](https://user-images.githubusercontent.com/93877359/152501492-3be8e8db-8022-4313-8a34-8120533e0d35.png)

**Comment by Mayome** [comment]
Sorry to resurrect this a little bit, but wanted to ask before I spend time setting up outlook and calendar syncs for nothing - the Win 10 agenda view is not functional as of now? So if I add synced calendars from elsewhere to outlook account, they will not show up in the flyout?

(I know the flyout suggests putting it in the Calendar app, but given that MS wants to throw it out as well this year, I would like to anticipate that change, but maybe that's for naught)

**Comment by Harv-st** [comment]
Also really interested in that, having Windows 11 TaskBar with Windows 10 Calendar would be a nice addition to the project.

**Comment by SHIELD22** [comment]
Im Windows 11 24H2 OD BUILD  26100.2314 Now the comand in all the other Previous version of windows where taken Out for the window 10 FLYOUT "CLOCK And Calendar  Now untill they Relased this version you have to go check you're Old setting's in 24H2 When All was working and what they deleted and took out in Regedit thats a start  check all you're Manual How to Add that back I've check gpedit.msc – Local Group Policy Editor.

 and therss' No setting's for the In.
The windows version Proc setting's there was that where it was removed .for the funtion of the windows 10 Full Task bar for previsons verions- this not gonna slow down are computes like the Myth Microsoft keeps telling us this has no barings on this what so ever this all part of the Classic veiw Setting's that went whith those verion they can be added back programed in like look at all setting's of previson verions you're reedit nes to added back . if you can get a Hold Reg edit fucntions in the closing to add back thease features sssso they can run with Explorer Patch 10.

**Reply by SHIELD22** [reply]
You need to set the version and re write  it for windows 24H2 that part is left out I can see that when you look at the top of Explorer patch 10 it said not available In this version of windows you to Put that in currant vsion and add the version  so it will work with Previous Ver's of windows . any coder should known that.we have the previos verions turned on and added now the script's to work with them need to also added to trigger the setting's to function propperly. you use the same 

Add-MpPreference -ExclusionPath "C:\Program Files\ExplorerPatcher"
Add-MpPreference -ExclusionPath "$env:APPDATA\ExplorerPatcher"
Add-MpPreference -ExclusionPath "C:\Windows\dxgi.dll"
Add-MpPreference -ExclusionPath "C:\Windows\SystemApps\Microsoft.Windows.StartMenuExperienceHost_cw5n1h2txyewy"
Add-MpPreference -ExclusionPath "C:\Windows\SystemApps\ShellExperienceHost_cw5n1h2txyewy"


find the shell experience host and the compatible etc it need to look for . and at to you're patch .

**Reply by SHIELD22** [reply]
they Put more fixes i on the patch tuesday n to funtion maybe you need to ad the 3rd party that works with it

**Reply by SHIELD22** [reply]
iin the flyout at the top of the calandar  the Large number's with the secs are not being displayed  the tiny add addiontional clocks are. Not the larger one .

**Reply by SHIELD22** [reply]
also check the previos power shell windows 10 Host_cwr   the one that came with winows cause it's the path it needs to go to where it stores those setting's

**Reply by SHIELD22** [reply]
_cw5

**Comment by SHIELD22** [comment]
https://raw.githubusercontent.com/EnziTheViking/PrettyWinUtil/main/winutil.ps1
Try this Chris is uselly good  at Tweeking the windows system registry to the Old styles working there something 'missing" use it as a reference guide . it might help

**Comment by SHIELD22** [comment]
{
        "Path": "HKLM:\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Explorer\\FlyoutMenuSettings",
        "Name": "ShowHibernateOption",
        "Type": "DWord",
        "Value": "0",
        "OriginalValue": "1"
      }
    ],
    "InvokeScript": [
      "powercfg.exe /hibernate off"
    ],
    "UndoScript": [
      "powercfg.exe /hibernate on"
    ]
  },

**Comment by SHIELD22** [comment]
there in  there I added a long time ago just trying to find the command settings to add.

**Comment by SHIELD22** [comment]
{
        "Name": "DevicesFlowUserSvc_*",
        "StartupType": "Manual",
        "OriginalType": "Manual"    see it it was Hidden from "services"

**Comment by SHIELD22** [comment]
This is the triky one Microsft messed up

 "InvokeScript": [
      "
        $TeamsPath = [System.IO.Path]::Combine($env:LOCALAPPDATA, ''Microsoft'', ''Teams'')
        $TeamsUpdateExePath = [System.IO.Path]::Combine($TeamsPath, ''Update.exe'')

        Write-Host \"Stopping Teams process...\"
        Stop-Process -Name \"*teams*\" -Force -ErrorAction SilentlyContinue

        Write-Host \"Uninstalling Teams from AppData\\Microsoft\\Teams\"
        if ([System.IO.File]::Exists($TeamsUpdateExePath)) {
            # Uninstall app
            $proc = Start-Process $TeamsUpdateExePath \"-uninstall -s\" -PassThru
            $proc.WaitForExit()
        }

        Write-Host \"Removing Teams AppxPackage...\"
        Get-AppxPackage \"*Teams*\" | Remove-AppxPackage -ErrorAction SilentlyContinue
        Get-AppxPackage \"*Teams*\" -AllUsers | Remove-AppxPackage -AllUsers -ErrorAction SilentlyContinue

        Write-Host \"Deleting Teams directory\"
        if ([System.IO.Directory]::Exists($TeamsPath)) {
            Remove-Item $TeamsPath -Force -Recurse -ErrorAction SilentlyContinue
        }

        Write-Host \"Deleting Teams uninstall registry key\"
        # Uninstall from Uninstall registry key UninstallString
        $us = (Get-ChildItem -Path HKLM:\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall, HKLM:\\SOFTWARE\\Wow6432Node\\Microsoft\\Windows\\CurrentVersion\\Uninstall | Get-ItemProperty | Where-Object { $_.DisplayName -like ''*Teams*''}).UninstallString
        if ($us.Length -gt 0) {
            $us = ($us.Replace(''/I'', ''/uninstall '') + '' /quiet'').Replace(''  '', '' '')
            $FilePath = ($us.Substring(0, $us.IndexOf(''.exe'') + 4).Trim())
            $ProcessArgs = ($us.Substring($us.IndexOf(''.exe'') + 5).Trim().replace(''  '', '' ''))
            $proc = Start-Process -FilePath $FilePath -Args $ProcessArgs -PassThru
            $proc.WaitForExit()

**Comment by SHIELD22** [comment]
Write-Host \"Install OneDrive\"
      Start-Process -FilePath winget -ArgumentList \"install -e --accept-source-agreements --accept-package-agreements --silent Microsoft.OneDrive \" -NoNewWindow -Wait
      "
    ]
  },
  "WPFTweaksDisableNotifications": {
    "Content": "Disable Notification Tray/Calendar",
    "Description": "Disables all Notifications INCLUDING Calendar",
    "category": "z__Advanced Tweaks - CAUTION",
    "panel": "1",
    "Order": "a024_",
    "registry": [
      {
        "Path": "HKCU:\\Software\\Policies\\Microsoft\\Windows\\Explorer",
        "Name": "DisableNotificationCenter",
        "Type": "DWord",
        "Value": "1",
        "OriginalValue": "0"
      },
      {
        "Path": "HKCU:\\Software\\Microsoft\\Windows\\CurrentVersion\\PushNotifications",
        "Name": "ToastEnabled",
        "Type": "DWord",
        "Value": "0",
        "OriginalValue": "1"
      }
    ]
  },
  "WPFTweaksRightClickMenu": {
    "Content": "Set Classic Right-Click Menu ",
    "Description": "Great Windows 11 tweak to bring back good context menus when right clicking things in explorer.",
    "category": "z__Advanced Tweaks - CAUTION",
    "panel": "1",
    "Order": "a028_",
    "InvokeScript": [
      "
      New-Item -Path \"HKCU:\\Software\\Classes\\CLSID\\{86ca1aa0-34aa-4e8b-a509-50c905bae2a2}\" -Name \"InprocServer32\" -force -value \"\"
      Write-Host Restarting explorer.exe ...
      $process = Get-Process -Name \"explorer\"
      Stop-Process -InputObject $process


this one in reg endit they messed  with the Calender and Clock you have to the functions of this what to "add" 

"HKCU:\\Software\\Classes\\CLSID\\{86ca1aa0-34aa-4e8b-a509-50c905bae2a2
